api/specs/web-server/_products.py

```diff
@@ -0,0 +1,26 @@
+# pylint: disable=protected-access
+# pylint: disable=redefined-outer-name
+# pylint: disable=too-many-arguments
+# pylint: disable=unused-argument
+# pylint: disable=unused-variable
+
+
+from fastapi import APIRouter
+from models_library.api_schemas_webserver.product import ProductPriceGet
+from models_library.generics import Envelope
+from simcore_service_webserver._meta import API_VTAG
+
+router = APIRouter(
+    prefix=f"/{API_VTAG}",
+    tags=[
+        "products",
+    ],
+)
+
+
+@router.get(
+    "/price",
```

Comment:
Q: this is the price per credits. I am not completely sure I get what is meant here. this is a rate, not a price right?
why is it just called price? since it's for the payment gateway I guess that is the price to buy credits not the burning credit rate.. but still just price is a little bit misleading. what about ```credits-price``` for example?

Reply:
renamed